converter/src/main/java/gov/cms/qpp/conversion/InputStreamSupplierSource.java

```diff
@@ -1,6 +1,6 @@
 package gov.cms.qpp.conversion;
 
-import org.apache.commons.io.IOUtils;
+import gov.cms.qpp.conversion.util.MeasuredInputStreamSupplier;
 
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.IOException'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)